media/js/base/mozilla-monitor-button.js

```diff
@@ -62,7 +63,10 @@ if (typeof window.Mozilla === 'undefined') {
             buttonURL += '&deviceId=' + r.deviceId;
             buttonURL += '&flowBeginTime=' + r.flowBeginTime;
             buttonURL += '&flowId=' + r.flowId;
-            monitorButton.setAttribute('href', buttonURL);
+            // applies url to all buttons and adds cta position
+            for (var i=0;i<buttons.length;i++) {
```

Comment:
Nit spaces after `;` separators
```suggestion
            for (var i=0; i<buttons.length; i++) {
```